fe/fe-core/src/main/java/org/apache/doris/analysis/LiteralExpr.java

```diff
@@ -358,6 +358,7 @@ public static LiteralExpr getLiteralByMysqlType(int mysqlType, boolean isUnsigne
                 break;
             case 5: // MYSQL_TYPE_DOUBLE
                 literalExpr = LiteralExpr.create("0", Type.DOUBLE);
+                literalExpr.setType(Type.DOUBLE);
```

Comment:
if double should set it type, should we just change `DOUBLE` branch in `LiteralExpr::create`?

Reply:
LiteralExpr::create is used in many places, it's best not to change it.